src/app/pages/user-management/user-list/user-list.component.ts

```diff
@@ -112,7 +112,7 @@ export class UserListComponent implements OnInit {
           err.message,
           "Could not update user's two-factor authentication status"
```

Comment:
Could you maybe just replace this with 'Could not ... user\'s ...' ? this would avoid the disabling of the linter and make the code a bit cleaner? don't know what they view on this is @egeldenhuys ?

Reply:
Prettier changes 'could not ..user\'s..' with "could not ... user's..." and does not seem to be configurable, so we disable the linter for that line instead.